docker/container-test.sh

```diff
@@ -33,3 +33,14 @@ function test_container {
     # Return proper exit code.
     [[ ${RESULT} == '"success"' ]]
 }
+
+function test_javahome {
+    # Check if JAVA_HOME works
+    docker run --rm "${CONTAINER_NAME}" /bin/bash -c '$JAVA_HOME/bin/java -version' &> /dev/null
```

Comment:
The variable is `CONTAINER_ID`

Reply:
the container instance with `$CONTAINER_ID` will be stopped right before `test_presto_starts()` is executed. this command here is creating a new container and remove it immediately after docker command execution.

If you suggest running two tests `test_javahome` and `test_presto_starts` within a single container instance, I can refactor it later.

Reply:
Sorry, I saw an error that the variable wasn’t set and got it mixed up. Your fix is correct.